### ISSUE TITLE
vscale_border doesn't work in the cores when vscale_mode=1

### ISSUE THREAD

**Issue by dfilskov** [MiSTer-devel/Main_MiSTer#347]
It seems vscale_border always works in the menu (even vscale_border=200 seems to work) but not inside the cores themselves when integer scale is used (vscale_mode=1).

**Comment by sorgelig** [comment]
Actually it's not a bug. These options are mutually exclusive. Border option won't give any benefit to integer scaling. It only can further decrease multiplier and make video even smaller which is not desired originally.

**Comment by dfilskov** [comment]
Ah - yes - I understand.

I guess it could be considered letting vscale_border work even when vscale_mode is not 0 ... after all even when you use integer scaling (e.g. vscale_mode=1) some cores' video height could match your vertical resolution edge-to-edge and therefore the image will be cropped if your monitor doesn't display all the pixels. However ... if your screen crops the video siginal there's no point it turning on vscale_mode=1 since the image will be scaled - and thus blurred - by the monitor :)

Another argument for letting vscale_border work even with vscale_mode=1 is to perhaps make it easier for users to get horizontal integer scaling. But I'm not really sure it helps :)

**Comment by sorgelig** [comment]
Scaler doesn't support cropping, therefore any scales of borders will work within the displayed resolution and both options will make the video smaller without crops. Border doesn't cut the video as well.
I think you treat it completely differently than it's designed for.

**Comment by dfilskov** [comment]
> Scaler doesn't support cropping / Border doesn't cut the video as well.

Yes - I meant the monitor will crop. The good reason for vscale_border is to see the whole image in spiste of the monitor's cropping (to keep the image inside the "safe area").

> any scales or borders will work within the displayed resolution and both options will make the video smaller without crops.

Sure - that's how it's supposed to be.

> I think you treat it completely differently than it's designed for.

What I meant was that even people who use integer scaling (vscale_mode=1) might have a monitor that crops the image (hides the edge of the output) and therefore they would need to use vscale_border (but that wouldn't work) ... but instead they can of course simply turn off integer scaling (vscale_mode=0) since if their monitor crops the image the monitor will have scaled it (to become larger than the screel hiding its edges) and therefore loose any sharpness that the intereger mode (vscale_mode=1) proved anyhow :) - I hope that makes sense. I short I mean that even though it could seem like a bug it probably isn't an issue as long as people with screens that up-scale / crops the MiSTer image turn off integer scaling.

Just close the image if you think that's best.
